src/hooks/useWindowDimensions/index.ts

```diff
@@ -13,28 +14,28 @@ const tagNamesOpenKeyboard = ['INPUT', 'TEXTAREA'];
 const isMobile = Browser.isMobile();
 
 /**
- * A convenience wrapper around React Native's useWindowDimensions hook that also provides booleans for our breakpoints.
+ * A wrapper around React Native's useWindowDimensions hook.
  */
 export default function (useCachedViewportHeight = false): WindowDimensions {
-    const {isFullScreenRef, lockedWindowDimensionsRef, lockWindowDimensions, unlockWindowDimensions} = useContext(FullScreenContext) ?? {
+    const {isFullScreenRef, lockedResponsiveLayoutResultRef, lockResponsiveLayoutResult, unlockResponsiveLayoutResult} = useContext(FullScreenContext) ?? {
         isFullScreenRef: useRef(false),
-        lockedWindowDimensionsRef: useRef<WindowDimensions | null>(null),
-        lockWindowDimensions: () => {},
-        unlockWindowDimensions: () => {},
+        lockedResponsiveLayoutResultRef: useRef<ResponsiveLayoutProperties | null>(null),
```

Comment:
Was it necessary to rename the variable names? I think it could be quite confusing `src/hooks/useWindowDimensions/index.ts`. wdyt? :)